resources/lang/pt-br/artist.php

```diff
@@ -22,13 +22,16 @@
     'page_description' => 'Artistas em destaque no osu!',
     'title' => 'Artistas em destaque',
 
+    'admin' => [
+        'hidden' => 'ARTISTA ATUALMENTE OCULTO',
+    ],
     'beatmaps' => [
         '_' => 'Beatmaps',
         'download' => 'Baixar modelo de beatmap',
         'download-na' => 'Modelo indisponível no momento',
     ],
     'index' => [
-        'description' => 'Artistas em destaque são artistas com quem estamos trabalhando em colaboração para que possamos trazer músicas novas e originais para o osu!. Esses artistas e suas músicas são escolhidos a dedo pela equipe do osu! pelas suas incríveis capacidades musicais e capacidade de mapeamento. Alguns desses artistas também criaram novas músicas exclusivas para uso no osu!.<br><br>Todas as músicas presentes nessa seção são fornecidas como arquivos .osz previamente temporizados e foram oficialmente licenciadas para uso no osu! e qualquer conteúdo relacionado ao mesmo.',
+        'description' => 'Artistas em destaque são artistas com quem estamos trabalhando em colaboração para que possamos trazer músicas novas e originais para o osu!. Esses artistas e suas músicas são escolhidas a dedo pela equipe do osu! e por sua ótima viabilidade no editor . Alguns desses artistas também criaram novas músicas exclusivas para uso no osu!.<br><br>Todas as músicas presentes nessa seção são fornecidas como arquivos .osz com BPM e Offset pré-disponibilizados e foram oficialmente licenciadas para uso no osu! e qualquer conteúdo relacionado.',
```

Comment:
Not really unneeded space. If I only say it's "viable", people would ask themselves where it is viable. And for the sake of consistency, as it states in the original english file, it says it's viable in the osu!editor.

I guess "previamente temporizados" wouldn't really specify where it is timed. It is, in english, because people use that expression a lot while editing, but confusing in portuguese. I'd rather specify it a bit more than probably leting some people confused.